src/content/en/fundamentals/progressive-web-apps/prpl.md

```diff
@@ -0,0 +1,242 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: 
+
+{# wf_updated_on: 2016-09-28 #}
+{# wf_published_on: 2016-09-28 #}
+
+# The PRPL Pattern {: .page-title }
+
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+Caution: PRPL is a pattern we believe makes sense for applications that can be
```

Comment:
I've moved it back to Dogfood and will have a discussion with the DevSite team about usage of the Dogfood tag.